docs/source/format/CanonicalExtensions.rst

```diff
@@ -72,4 +72,66 @@ same rules as laid out above, and provide backwards compatibility guarantees.
 Official List
 =============
 
-No canonical extension types have been standardized yet.
+Fixed shape tensor
+==================
+
+* Extension name: `arrow.fixed_shape_tensor`.
+
+* The storage type of the extension: ``FixedSizeList`` where:
+
+  * **value_type** is the data type of individual tensor elements.
+  * **list_size** is the product of all the elements in tensor shape.
+
+* Extension type parameters:
+
+  * **value_type** = Arrow DataType or Field of the tensor elements.
```

Comment:
```suggestion
  * **value_type** = the Arrow data type of individual tensor elements.
```

Removing the mention to Field here as well? (like it was removed from the equivalent item above describing the FixedSizeList)